share/governance.md

```diff
@@ -148,12 +155,39 @@ role. Appointments last until either the officer steps down, or the unlikely
 and unfortunate situation where the PMC removes the officer. The PMC can add
 more officer roles as it sees fit by modifying this document.
 
-1) Election Officer: The election officer can be on the PMC, as long as they are not running in the election they are officiating
-  * The election officer will be a member of the `election-officer` GitHub group which grants him write access to the plumbing repository.
+1) Election Officer: The election officer can be on the PMC, as long as they are not running in the election they are officiating.
+  * The election officer will be a member of the `election-officer` GitHub group which grants them write access to the plumbing repository.
+
+2) Security Officer: This is the point of contact for external or internal security issues, this person has a published GPG key, and will be the main point of contact for CVE numbers and such.
+
+3) Communications Officer: This is the main point of contact for external and internal publicity and marketing efforts and requests.
+
+### Subcommittees
+
+Each subcommittee will have at least one member who is also on the PMC and at
+least three members in total. To be elligible for a subcommittee, one must
+already be Collaborator.
+
+#### Module committee
+
+This committee will manage what Vox Pupuli has generally managed including
+Puppet modules, puppet-lint and plugins, editor integrations, the associated
+plumbing to test and manage these repositories and the like.
+
+#### Security committee
+
+This committee will manage security incidents including working with third
+parties such as Perforce.
+
+#### Language committee
+
+This committee will work with the community and Perforce to create standards
```

Comment:
@binford2k highlighted this elsewhere and the actual committee would include Vox Pupuli, Perforce, community members and representatives from other companies that give paid support. This subcommittee is meant to identify who would represent Vox Pupuli.

Reply:
I agree, but the Language Reference should be wholly owned by the OSS project/not-for-profit and not a business making money on Puppet to prevent a takeover/messy divorce situation.

Reply:
Good point. However that comes to be, Vox Pupuli will be involved, so this is to capture who those people are representing VP.

Reply:
the steering committee should never have a majority or veto power or anything such by any single for-profit company. It should always be community forward. That said, most of the people in our community do represent a company to one degree or another. Even myself (Overlook) and @bastelfreak (Betadots) work at least partially on behalf of companies.

As long as decision making leadership is shared equitably, it shouldn't really be an issue. If we want to be cautious about that, we could differentiate between company seats and personal seats and require attestations that people represent themselves only.

Reply:
This section is just meant to convey who will represent VP. Make up of the PMC and subcommittees is being discussed below under Subcommittees heading.

Reply:
I think PostgreSQL has some rules about not too many people from the same company in certain positions and I like that. I recall they had to change things when people moved to a different company or an acquisition, but on my phone now and it's hard to find the reference